### PR TITLE
Fixed bug on setDiscount()

### PR DESCRIPTION
**Pull request by poolpitako**

### CODE REVIEW

File: contracts/Strategy.sol [hunk 1]
```diff
@@ -186,21 +175,29 @@ contract Strategy is BaseStrategyInitializable {
...
         internal
         override
         returns (uint256 _liquidatedAmount, uint256 _loss)
     {
-        uint256 freeAssets = balanceOfWant();
-        _liquidatedAmount = freeAssets;
+        _liquidatedAmount = balanceOfWant();
+
+        if (_amountNeeded > _liquidatedAmount) {
+            _loss = _amountNeeded.sub(_liquidatedAmount);
```
Comment (fp-crypto) [inline contracts/Strategy.sol:191]: Is there a reason to take a loss here? The funds shouldt be held as stkAave temporarily?
